versioned_docs/version-23.10/integrations/data-analytics/sc-canopsis-events.md

```diff
@@ -0,0 +1,417 @@
+---
+id: sc-canopsis-events
+title: Canopsis Events
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Canopsis Events stream connector allows you to send data from Centreon to Canopsis
+using their HTTP REST API.
+
+## Before starting
+
+- In most cases, you will want to send data from the central server. It is also possible 
+to send it from a remote server or a poller (e.g. if you want to avoid the central 
+server being a SPOF, or if you are an MSP and you install the stream connector on a 
+poller or a remote server within your customer's infrastructure).
+- By default, the Canopsis Events stream connector sends events from 
+**[host_status](https://docs.centreon.com/fr/docs/developer/developer-broker-mapping/#host-status)**, 
+**[service_status](https://docs.centreon.com/fr/docs/developer/developer-broker-mapping/#service-status)** 
+and **[acknowledgement](https://docs.centreon.com/fr/docs/developer/developer-broker-mapping/#acknowledgement)** 
+Broker events. The event format is shown **[here](#event-format)**.
+- These events are sent each time a host or a service is checked. Various parameters let 
+you [filter out events](#filtering-or-adapting-the-data-you-want-to-send-to-canopsis).
+
+## Compatibility
+
+This stream connector is designed to be compatible with Canopsis' API v.4, and more specifically the following versions of Canopsis : 22.10, 
+23.04, 23.10 and 24.04.
+
+## Installation
+
+Perform the installation on the server that will send data to Canopsis (central server, 
+remote server, poller).
+
+1. Login as `root` on the Centreon central server using your favorite SSH client.
+
+2. Run the following command:
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+```shell
+dnf install centreon-stream-connector-canopsis
+```
+
+</TabItem>
+
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```shell
+dnf install centreon-stream-connector-canopsis
+```
+
+</TabItem>
+
+<TabItem value="Debian 11" label="Debian_11">
+
+```shell
+apt install centreon-stream-connector-canopsis
+```
+
+</TabItem>
+</Tabs>
+
+## Configuring Canopsis
+
+You will need to configure your Canopsis host to receive data from Centreon. Refer to the 
+[Canopsis user guide](https://doc.canopsis.net/guide-utilisation/menu-administration/droits/)
+, in particular check if creation, reading and deletion rights are activated. See the following documentations: 
+* [rights administration](https://doc.canopsis.net/guide-utilisation/menu-administration/droits/) 
+* [planification administration] (https://doc.canopsis.net/guide-utilisation/menu-administration/planification/) (in case of downtimes). 
```

Comment:
```suggestion
* [planification administration](https://doc.canopsis.net/guide-utilisation/menu-administration/planification/) (in case of downtimes). 
```